ddtrace/internal/rand.py

```diff
@@ -0,0 +1,9 @@
+from ddtrace import compat
+
+import pyximport; pyximport.install()
+
+try:
+    from ._rand import rand64bits
```

Comment:
when would this fail? do we need a fallback or should we just fail?

Reply:
Given that we're now enforcing c-extensions then we don't need a fallback here.